orgs-using-d.dd

```diff
@@ -108,16 +108,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING job.oschina.net/position/792_118197_21895)
         )
     )
-
-    $(COMMENT
-    $(DORG Remedy Games, http://remedygames.com, remedy.svg,
+    $(DORG Remedy Entertainment, http://remedygames.com, remedy.png,
```

Comment:
we have more space here, so the long name would fit. However should I use "Remedy Games" for consistency?